Domain/FinancialYearConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Domain;
+
+public class FinancialYearConfiguration
```

Comment:
This shouldn't be an aggregate root I think? It should just be configuration which is retrieved wherever needed using the .NET `IOptions` pattern